python/python/glide/async_commands/core.py

```diff
@@ -733,6 +733,36 @@ async def mset(self, key_value_map: Mapping[str, str]) -> TOK:
             parameters.extend(pair)
         return cast(TOK, await self._execute_command(RequestType.MSet, parameters))
 
+    async def msetnx(self, key_value_map: Mapping[str, str]) -> bool:
+        """
+        Sets multiple keys to values if the key does not exist. The operation is atomic, and if one or
+        more keys already exist, the entire operation fails.
+
+        Note:
+            When in cluster mode, all `keys` must map to the same hash slot.
```

Comment:
Since keys isn't an argument for this function
```suggestion
            When in cluster mode, all all keys in `key_value_map` must map to the same hash slot.
```

Reply:
thanks!